app/views/items/_item_links.html.erb

```diff
@@ -1,10 +1,10 @@
+<%if user_signed_in?%>
+  <%if policy(item).create?%>
+  <div class="links">
 
-<%if policy(item).create?%>
-<div class="links">
-
-    <%= link_to 'Show', item_path(item) %>
-    <%= link_to 'Edit', edit_item_path(item) %>
-    <%= link_to 'Destroy', item_path(item), method: :delete, data: { confirm: 'Are you sure?' } ,remote: true,class:"delete"%>
-
-</div>
+      <%= link_to 'Show', item_path(item) ,style:'color:blue'%>
+      <%= link_to 'Edit', edit_item_path(item) ,style:'color:green'%>
+      <%= link_to 'Destroy', item_path(item), method: :delete, data: { confirm: 'Are you sure?' } ,style:'color:red',remote: true,class:"delete"%>
```

Comment:
I have been reminding you since last month.
**Don't Use Inline styling.**

Reply:
Done